dist/source/repo-creator/sponsorship.html

```diff
@@ -0,0 +1,40 @@
+<template>
+	<require from="./choose-repository.css"></require>
```

Comment:
This page should have its own CSS file.  Long term, there are plans for revamping this UI to be better looking, but for now it is reasonable to copy over the necessary parts into a new file `my-sponsorships.css`.  When you do, try to remove any CSS selectors that aren't being used in this page (e.g., `searchBtn`).